### PR TITLE
Vec<T> is pronounced 'vec'

### PR DESCRIPTION
**Pull request by Manishearth**
I've never heard it pronounced "vector". Is this an outdated recommendation?

(or have I been doing it wrong all this time)

r? @steveklabnik

### CODE REVIEW

**Comment by frewsxcv** [comment]
https://github.com/rust-lang/rust/search?utf8=%E2%9C%93&q=vector&type=

**Comment by Manishearth** [comment]
Right, but that's _written as_ "vector". A `Vec` _is_ a vector, just like an apple is a fruit. You don't say "fruit" when you see the word "apple". The question is how Vec is pronounced when in code _written as `Vec`_

**Comment by frewsxcv** [comment]
> The question is how Vec is pronounced when in code _written as `Vec`_

i personally do say 'vector'. though i'm apathetic about whether we encourage that or not

**Comment by kennytm** [comment]
If the point is to introduce the word "vector", we would reword the sentence keep mentioning "vector" but eliminate the "pronunciation".

> A contiguous growable array type with heap-allocated contents, **commonly known as “vector”** or “dynamic array”. **In Rust this type is written as `Vec<T>`**.

**Comment by shepmaster** [comment]
FWIW, I'm a big fan of things describing how they are pronounced. (Like `'a` is "tick a"). Adding some IPA to show how `Vec` is *actually pronounced* wouldn't make me sad 😺 .

**Comment by steveklabnik** [comment]
It's both really, I'd say "vek t" for `Vec<T>` but also sometimes say "a vector."

**Comment by Manishearth** [comment]
>  also sometimes say "a vector."

While reading code? For "a vector" my comment about "a fruit" applies 😄

**Comment by alexcrichton** [comment]
ping @steveklabnik for review!

**Comment by est31** [comment]
so this chapter should be renamed to "`vec`s"? https://doc.rust-lang.org/book/vectors.html

**Comment by frewsxcv** [comment]
> so this chapter should be renamed to "`vec`s"? https://doc.rust-lang.org/book/vectors.html

why?

**Comment by Manishearth** [comment]
Again, this doesn't make it not a "vector". It just means that you _pronounce_ "`Vec<T>`" as "vec" in code. It's still a vector.

**Comment by est31** [comment]
I see, thanks!
